eng/pipelines/templates/steps/common.yml

```diff
@@ -8,6 +8,10 @@ steps:
     parameters:
       OSName: $(OSVmImage)
 
+  - template: eng/pipelines/templates/scripts/verify-path-length.yml@azure-sdk-tools
```

Comment:
Are we doing this in all pipelines in the other languages? I thought we were only doing it in the analyze step.